src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -7091,6 +7095,32 @@ private static void processP(Element node, TagStatus status) {
       FightRequest.logText(str, status);
     }
 
+    if (KoLCharacter.hasEquipped(ItemPool.TOY_CUPID_BOW)) {
```

Comment:
This is identical to checking `status.cupidbow`, which is what you check down below. Why are these different?